index.js

```diff
@@ -1111,14 +1111,16 @@ instance.prototype.action = function(action) {
 			break;
 
 		case 'custom':
-			var hexData = opt.custom.replace(/\s+/g, '');
-			var tempBuffer = new Buffer(hexData, 'hex');
-			cmd = tempBuffer.toString('binary');
-
-			if ((tempBuffer[0] & 0xF0) === 0x80) {
-				self.sendVISCACommand(cmd);
-			} else {
-				self.log('error', 'Error, command "' + opt.custom + '" does not start with 8');
+			if (typeof opt.custom === 'string' || opt.custom instanceof String) {
```

Comment:
@krocheck What about logging an error when the opt.custom is not a string instead of just silently ignoring it?